packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss

```diff
@@ -134,38 +141,41 @@
       display: none;
     }
 
-    padding-left: units(
-      $theme-site-margins-mobile-width - $theme-alert-bar-width
-    );
-    @include at-media($theme-site-margins-breakpoint) {
-      padding-left: 2 * $alert-icon-optical-padding;
+    .usa-alert__heading,
+    .usa-alert__text {
+      margin-left: 0;
+
+      @include at-media($theme-site-margins-breakpoint) {
+        margin-left: 0;
+      }
     }
   }
 }
 
 // Variant - slim
 @mixin add-slim-alert-icon {
   &:before {
-    background-size: $alert-slim-icon-size;
-    height: $alert-slim-icon-size;
-    top: units($theme-alert-padding-y) * 0.5;
-    width: $alert-slim-icon-size;
     @supports (mask: url("")) {
-      mask-size: $alert-slim-icon-size;
+      mask-size: units($alert-slim-icon-size);
     }
+    height: units($alert-slim-icon-size);
+    width: units($alert-slim-icon-size);
+    top: units($alert-slim-icon-padding);
   }
 }
 
 @mixin alert-styles-slim {
   .usa-alert__body {
-    @include u-padding-y(1);
+    @include u-padding-y($alert-slim-icon-padding);
     @include add-slim-alert-icon;
 
-    padding-left: $alert-slim-icon-size + $alert-icon-optical-padding;
-    @include at-media($theme-site-margins-breakpoint) {
-      padding-left: calc(
-        units($theme-site-margins-width) + $alert-slim-icon-size
-      );
+    .usa-alert__heading,
+    .usa-alert__text {
+      margin-left: $alert-x-align-text-with-slim-icon;
+
+      @include at-media($theme-site-margins-breakpoint) {
+        margin-left: $alert-x-align-text-with-slim-icon;
+      }
```

Comment:
**Question:** Same as before, do we need this media query if we're applying the same margin as the line above?

Reply:
Good catch - removed the redundant/unnecessary media queries in [8e5295c](https://github.com/uswds/uswds/pull/5636/commits/8e5295cd51d7fdd232d96f514025210d6d987b5f)